src/libraries/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -97,9 +104,8 @@ internal static class DispatchProxyGenerator
             }
         }
 
-        // Unconditionally generates a new proxy type derived from 'baseType' and implements 'interfaceType'
```

Comment:
Why is this being removed?

Reply:
This is is not resolved.

Reply:
Hmmm... I added it back. Is that not the case?